Automattic/wp-calypso
# State: Convert getDiscountedRawPrice to TypeScript

## Proposed Changes

This converts the `getDiscountedRawPrice()` function (and the associated `getPlanDiscountedRawPrice` and `getSitePlanRawPrice` and `getPlanRawDiscount`) to TypeScript and clarifies the meaning of its third parameter that converts the price to a monthly one (renaming it from `isMonthly` to `returnMonthly`). 

The monthly conversion operation is inherently imprecise because it performs math on floating point numbers and will certainly result in rounding errors in some currencies.

## Testing Instructions

Verify that there are no places still using the `isMonthly` option when calling any function (there's a standalone _function_ called `isMonthly`, but that's separate).

You can also verify that the "per month" prices on the `/plans` page for annual plans look the same both before and after this change:

<img width="1212" alt="after-rename" src="https://user-images.githubusercontent.com/2036909/229560188-277db490-7001-4bc0-8e57-60257cfa6440.png">
